### DISCUSSION TITLE
Received marketing message at my GitHub email

### DISCUSSION THREAD

**Discussion by rossng** [the-guild-org/the-guild-website#1358] [Q&A] [answered]
I recently received unsolicited mail from The Guild at my GitHub email address. I didn't give consent for this, so why did it happen?

@Urigo

**Comment by Urigo** [comment, marked as answer]
I'm sorry @rossng, I've stopped and cancelled this email thing (new software) in the middle of it while I'm trying to figure out why this is happening

**Reply by rossng** [reply]
Thank you, I appreciate the fast response.
